2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
```

Comment:
@mfc I'm wondering if I chose the category correctly. Wouldn't the article category be more appropriate?

Reply:
You can use both:

```
categories:
- talks
- articles
```

Reply:
```suggestion
categories:
- talks
- articles
```